setup.sh

```diff
@@ -60,6 +64,22 @@ else
 	usage 1
 fi
 
+# make sure that scram is available
+# examples:
+#   in a non-interactive Docker container this will evaluate to "hB"
+#   in an login session this will evaluate to "himBCHP"
+if [[ $- != *i* ]]; then
+	echo "WARNING: Non-interactive shell found!"
+    if [[ -f "${HOME}/.bashrc" ]]; then
```

Comment:
use tabs not spaces (everywhere in this file)

Reply:
Fixed